app/admin/models/Locations_model.php

```diff
@@ -65,9 +64,8 @@ class Locations_model extends AbstractLocation
         'gallery' => ['multiple' => TRUE],
     ];
 
-    protected static $allowedSortingColumns = [
+    public static $allowedSortingColumns = [
         'distance asc', 'distance desc',
```

Comment:
Tracking back on this, let's use a new method to overload the value instead of making it public to minimise breaking changes.

Reply:
Good idea - see here: